### PR TITLE
update front page

### PR DESCRIPTION
**Pull request by npentrel**
Code samples for components and services sections are not fully tested - otherwise good for review.

### CODE REVIEW

**Comment by sguequierre** [comment]
I think this looks excellent. The only thing I can think of is if we want to highlight navigation at all, that's missing in this draft

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -2054,3 +2054,319 @@ h3.body-header[style] {
...
+  justify-content: space-between;
+}
+
+.hero-text {
+  min-width: 50%;
+  max-width: 50%;
+}
+
+.hero-text > h1 {
+  margin-top: 7rem;
```
Comment (npentrel) [inline assets/scss/_styles_project.scss:2104]: Should change this for mobile

File: docs/_index.md [hunk 2]
```diff
@@ -11,110 +11,770 @@ sitemap:
...
+
+  </div>
+</div>
+</div>
+
+{{% /tab %}}
+{{% tab name="Data Management" %}}
+<div class="tabcontent">
+
+```json
```
Comment (npentrel) [inline docs/_index.md:366]: Replace with screenshot

File: assets/scss/_styles_project.scss [hunk 3]
```diff
@@ -2054,3 +2054,319 @@ h3.body-header[style] {
...
+
+.front-card-container > .hover-card, .front-card-container > .hover-card > a {
+  min-width: unset;
+  max-width: unset;
+  text-align: center;
+  margin-left: unset;
+  align-content: center;
+}
+
+.front-card-container > .hover-card.primary, .front-card-container > .cta.secondary {
```
Comment (npentrel) [inline assets/scss/_styles_project.scss:2074]: Margin/padding inside card too much especially for mobile

File: assets/scss/_styles_project.scss [hunk 4]
```diff
@@ -2066,3 +2066,309 @@ h3.body-header[style] {
...
+  padding: 0;
+  padding-top: 0.25rem;
+  border: unset;
+  flex-grow: 1;
+}
+
+.table.horizontalheaders > .tab-content > div > .tabcontent > picture > img, .table.horizontalheaders > .tab-content > div > .tabcontent > img {
+  flex-grow: 1;
+  margin-bottom: 0;
+  max-width: 500px;
```
Comment (npentrel) [inline assets/scss/_styles_project.scss:2305]: Doesn't work for mobile

Comment (npentrel) [inline assets/scss/_styles_project.scss:2305]:
```suggestion
  width: 500px;
  max-width: 100%;
```

File: docs/_index.md [hunk 5]
```diff
@@ -11,110 +11,931 @@ sitemap:
...
+            robot_part_id=main_part.id, num_log_entries=5)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="explanation">
+  <div class="explanationtext">
+
+Get status information and logs fromo all your deployed machines.
```
Comment (sguequierre) [inline docs/_index.md:844]:
```suggestion
Get status information and logs from all your deployed machines.
```

File: docs/_index.md [hunk 6]
```diff
@@ -11,110 +11,931 @@ sitemap:
...
+
+{{% /tab %}}
+{{% tab name="Observability" %}}
+
+<div class="tabcontent">
+
+{{< tabs >}}
+{{% tab name="Viam App" %}}
+
+{{< imgproc src="/fleet/dashboard.png" alt="Dashboard view of machine status information" resize="1200x" class="imgzoom aligncenter" >}}
```
Comment (sguequierre) [inline docs/_index.md:814]: This image-- the black looks a bit weird right up on the top. Would be better centered more vertically. If it's too much to fix that no worries though.

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/_index.md (hunk 6 above)
Comment (npentrel) [reply docs/_index.md:813]: Added a bit of margin to the top - Adding more ends up looking weird and then we have to also make the image smaller :/

File: docs/_index.md [hunk 7]
```diff
@@ -11,110 +11,930 @@ sitemap:
   priority: 1.0
 outputs:
   - html
-carouselscript: true
+aliases:
+  - "/getting-started/"
```
Comment (npentrel) [inline docs/_index.md:15]:
```suggestion
  - "/get-started/"
  - "/getting-started/"
```

**Pull request merged**